pkg/types/ccipocr3/generic_types.go

```diff
@@ -83,18 +83,26 @@ func (c ChainSelector) String() string {
 }
 
 type CCIPMsg struct {
```

Comment:
This is going to be the `Any2Any` message type right? The type that holds the fields that are present on all `Any2{Chain}` and `{Chain}2Any` messages? If so lets make it clear that this type will hold all fields and is not chain specific?

Reply:
> This is going to be the Any2Any message type right?
Yeah, I wanted to avoid `Any2Any` prefix. It's just THE `CCIPMsg`.

We can name the others `CCIPMsgToChainX` or ``CCIPMsgFromChainX` to make them chain specific